tools/cicd_self_hosted_agents/example.auto.tfvars

```diff
@@ -0,0 +1,22 @@
+resource_group_name = "caf-ghr"
+location            = "Canada Central"
+tags = {
+  account_coding = "000000000000000000000000",
+  billing_group  = "LICENSE_PLATE", # License Plate
```

Comment:
I don't think we want these tags. These are tags that are set by our automation and we don't want to train people to set them.